apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -182,9 +183,10 @@ if (true) {
     })
   );
 }
-
+const calcomAdapter = CalComAdapter(prisma);
 export default NextAuth({
-  adapter: PrismaAdapter(prisma),
+  // @ts-ignore
```

Comment:
Required for the moment till we fully migrate our DB to use next-auth-prisma-adapter.

Reply:
Nice work on the custom calcomAdapter 👍🏻

Reply:
![](https://media.giphy.com/media/NCjISbEPFxm48/giphy.gif)